src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -241,14 +316,41 @@ export const onRequest = defineMiddleware(async (context, next) => {
 });
 ```
 
+##### `calledFrom`
+
+<p>
+
+**Type:** `"rpc" | "form"`
+</p>
+
+Whether an action was called using an RPC function or an HTML form action.
+
+##### `name`
+
+<p>
+
+**Type:** `string`
+</p>
+
+The name of the action. Useful to track the source of an action result during a redirect.
+
+##### `handler()`
+
+<p>
+
+**Type:** <code>() => Promise\<<a href="#saferesult">SafeResult</a>\></code>
+</p>
+
+A method allowing you to programmatically call the action to get the result.
```

Comment:
```suggestion
A method to programmatically call an action to get the result.
```
I *think* (though I didn't look at the broader context so I could be wrong) this is maybe to call *an/any* generic action? When you say *the* action, there must be one in question you're referring to.

It's much more common to use "a/an" unless you can definitively point to the specific thing this refers to in context. (Like, a config option 2 or 3 levels deep, where you're saying it refers to *the* main config option in question.) So, I might be wrong, but this is how I'd decide which to use here!